### PR TITLE
add signature utilities for verifying wallet address

### PR DESCRIPTION
**Pull request by jmilldotdev**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by jmilldotdev** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2280** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2280" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2280?utm_source=stack-comment).

File: src/utils/signature.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { verifyMessage } from '@ambire/signature-validator'
+import { JsonRpcSigner } from '@ethersproject/providers'
+import { readProvider } from 'constants/readProvider'
+
+export const verifyWallet = async (signer: JsonRpcSigner, message: string) => {
```
Comment (tomquirk) [inline src/utils/signature.ts:5]: **question** do we need this function? looks like it just wraps `signer.signMessage(message)`.

If so, perhaps a more representitive name might be `signMessage`

File: src/utils/signature.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  if (!signer || !message) {
+    return
+  }
+  const signature = await signer.signMessage(message)
+  return signature
+}
+
+export const verifySignature = async (
```
Comment (tomquirk) [inline src/utils/signature.ts:13]: **question** similar to above perhaps `verifyMessage` might be more clear?

Comment (jmilldotdev) [reply src/utils/signature.ts:13]: truuuu i guess there's basically no point to these utilities now that i look at them given they're such thin wrappers. maybe a future action item is to remove the ambire library as a dep and move its code into our codebase.

i just deleted the file for now.

**Comment by tomquirk** [comment]
@jmilldotdev just checking, this is in `draft` right?

**Comment by jmilldotdev** [comment]
yes it is in draft

**Comment by tomquirk** [comment]
Closing for now!

**Pull request closed without merging**